analyzer/protocol/stun/Readme.md

```diff
@@ -0,0 +1,58 @@
+# STUN
+
+Original logic can be found here:
+
+- https://github.com/r-franke/spicy_stun
+- https://github.com/r-franke/spicy_stun/issues/1
+
+Additional logic has been added on top of the original logic.
```

Comment:
Could you first put a block explaining what this analyzer does (e.g., "... adds an analyzer for the STUN protocol, see ... "). You could then call out that this is based on r-frankie/spicy-stun, but I would drop the "Additional logic has been added ..." part.

While you are at it, could you rename this file `README` most other files?

Reply:
I don't understand why this needs to be changed.  There are links to what STUN is.

Reply:
I changed some of the language here.

Reply:
Also, I like the .md because you have formatting for logs and things.

Reply:
"Additional logic has been added" part is to signify we used code, but it's not a straight up copy and paste to here.  On another analyzer in this repo Robin had me specify something similar to this where I copied portions and wrote portions, so that was my reasoning for that language.

Reply:
Outdated